packages/pymedphys_labs/setup.py

```diff
@@ -0,0 +1,61 @@
+from glob import glob
+from os.path import abspath, dirname, join as pjoin
+from setuptools import setup, find_packages
+
+root = dirname(abspath(__file__))
+
+
+def execfile(fname, globs, locs=None):
+    locs = locs or globs
+    exec(compile(open(fname).read(), fname, "exec"), globs, locs)
+
+
+source_path = 'src'
+packages = find_packages(source_path)
+root_packages = [
+    package
+    for package in packages
+    if "." not in package
+]
+
+assert len(root_packages) == 1
+package = root_packages[0]
+package_directory = pjoin(root, source_path, package)
+
+
+def get_variable_from_file(filepath, variable):
+    filepath_in_package = pjoin(package_directory, filepath)
+    globs = {}
+    execfile(filepath_in_package, globs)
+    variable_value = globs[variable]
+
+    return variable_value
+
+
+version = get_variable_from_file('_version.py', '__version__')
+install_requires = get_variable_from_file(
+    '_install_requires.py', 'install_requires')
```

Comment:
Or have you already done this and I'm failing in my review logic?

Reply:
This is certainly prepping to make it DRY. But it is not there yet.

Reply:
I figured we could have a period of time where it wasn't DRY, as it will take a bit of time to sort out the tools to DRY it all up. Bit of a chicken and the egg, need to have it in this format to know what tools are needed to DRY things.

Reply:
Fair enough.